packages/blaze/builtins.js

```diff
@@ -32,22 +32,41 @@ Blaze.With = function (data, contentFunc) {
   return view;
 };
 
+
+/**
+ * @summary Shallow compare of two bindings.
+ * @param {Binding} x
+ * @param {Binding} y
+ */
+function _isEqualBinding(x, y) {
+  return x && y ? x.error === y.error && x.value === y.value : x === y;
+}
+
 /**
  * Attaches bindings to the instantiated view.
  * @param {Object} bindings A dictionary of bindings, each binding name
  * corresponds to a value or a function that will be reactively re-run.
- * @param {View} view The target.
+ * @param {Blaze.View} view The target.
  */
 Blaze._attachBindingsToView = function (bindings, view) {
+  function setBindingValue(name, value) {
+    if (value instanceof Promise) {
```

Comment:
Usually it is better to check if the value is [thenable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#thenables) instead of an instance of promise. This is what js engines and other parts of Meteor do, and allows promise polyfills to work correctly.

Reply:
OK, done. I couldn't run tests locally, though, due to dependency issues @Grubba27.

Reply:
What do you mean? can you send me the issues that you had?

Reply:
Yeah, I thought it could work by updating tracker deps in the spacebar-tests. Will revert and make a new Blaze release by tomorrow.

Reply:
@radekmie @Grubba27 can you please take a look at https://github.com/meteor/blaze/pull/416 which should fix the tracker dependency issue by updating the `test-app` project to Meteor 2.12